regions/io/ds9/read.py

```diff
@@ -71,7 +71,11 @@ def parse_coordinate(string_rep, unit):
         Parse a single coordinate
         """
         # Any ds9 coordinate representation (sexagesimal or degrees)
-        if 'd' in string_rep or 'h' in string_rep:
+        if string_rep[-1] == 'r':
+            return coordinates.Angle(string_rep[:-1], unit=u.rad)
+        elif string_rep[-1] in ['i', 'p']:
```

Comment:
`# i, p = image, pixel`
(right?)

Reply:
i = image; p=physical according to specs

Reply:
so yeah!